### ISSUE TITLE
apply it in Chinese

### ISSUE THREAD

**Issue by HZNUJeffreyRen** [miaomiaomiao-LJY/ppg_tacotron#11]
I want to apply it in Chinese. I'm working on a data set in the same format as TIMIT. 
What do the first two numbers represent in the PHN file? Is it the timestamp of this phoneme?
![image](https://user-images.githubusercontent.com/65889515/169198999-2d83ae8f-d6b4-4edf-bc48-1b7800b42f55.png)

**Comment by LJY-M** [comment]
> I want to apply it in Chinese. I'm working on a data set in the same format as TIMIT. What do the first two numbers represent in the PHN file? Is it the timestamp of this phoneme? ![image](https://user-images.githubusercontent.com/65889515/169198999-2d83ae8f-d6b4-4edf-bc48-1b7800b42f55.png)

Yes, the first digit represents the time when the phoneme begins and the second digit represents the time when the phoneme ends.
